groups/wg-ux/groups.yaml

```diff
@@ -0,0 +1,17 @@
+# UX WG
+groups:
+  - email-id: ux@knative.team
+    name: ux
+    description: |-
+      UX WG email
+    settings:
+      WhoCanPostMessage: "ANYONE_CAN_POST"
+      ReconcileMembers: "true"
+    owners:
+      - zainabhusain@ocadu.ca
+      - mariana.mejia@ocadu.ca
+      - leoli@redhat.com
+      - cmurray@redhat.com
```

Comment:
/hold

@mmejia02 @zainabhusain227 @Leo6Leo @Cali0707 

Are these email addresses good?

Reply:
These seem to match the emails used on the calendar invite for the meetings so I think it's correct